projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-external-media-import.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * WordPress.com media import page.
+ *
+ * Adds WordPress.com-specific external media page to WordPress Media > Import.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+if ( empty( $_GET['wpcom_external_media_import_page'] ) ) { // phpcs:disable WordPress.Security.NonceVerification.Recommended
+	return;
+}
+
+/**
+ * Register the WordPress.com-specific external media page to Media > Import.
+ */
+function add_wpcom_external_media_import_page() {
+	$wpcom_external_media_import_page_hook = add_submenu_page(
+		'upload.php',
+		__( 'Import Media', 'jetpack-mu-wpcom' ),
+		__( 'Import Media', 'jetpack-mu-wpcom' ),
+		'upload_files',
+		'wpcom_external_media_import_page',
+		'render_wpcom_external_media_import_page'
+	);
+
+	add_action( "load-$wpcom_external_media_import_page_hook", 'enqueue_wpcom_external_media_import_page' );
+}
+add_action( 'admin_menu', 'add_wpcom_external_media_import_page' );
+
+/**
+ * Enqueue the assets of the wpcom external media page.
+ */
+function enqueue_wpcom_external_media_import_page() {
+	jetpack_mu_wpcom_enqueue_assets( 'wpcom-external-media-import-page', array( 'js' ) );
+}
+
+/**
+ * Render the container of the wpcom external media page.
+ */
+function render_wpcom_external_media_import_page() {
+	$title                  = __( 'Import Media', 'jetpack-mu-wpcom' );
+	$description            = __( 'WordPress allows you to import media from various platforms directly into the Media Library. To begin, select a platform from the options below:', 'jetpack-mu-wpcom' );
```

Comment:
Nit: should this be `WordPress.com` 🙈 
```suggestion
	$description            = __( 'WordPress.com allows you to import media from various platforms directly into the Media Library. To begin, select a platform from the options below:', 'jetpack-mu-wpcom' );
```

Reply:
👍